### DISCUSSION TITLE
2024.09.11.1039

### DISCUSSION THREAD

**Discussion by guanlisheng** [moneymanagerex/android-money-manager-ex#1825] [Announcements]
## What's Changed
Features:
- Add financial year support in reports and transaction lists.
- Unify Deposit/Withdrawal AS transfer, not counted in Income vs Expenses.
- Refine Notification & Autopost switch logic.

Refactors:
- Moved DBPref into SharedPreference for unified preference management.
- Upgrade JDK17

Bug Fixes:
- Minor cleanups across various files and logic.
- Updated minSdkVersion to 23.

Doc Updates:
- Updated documentation for nested categories, recurring transactions, and JDK17 reference.

**Full Changelog**: https://github.com/moneymanagerex/android-money-manager-ex/compare/2024.08.25.1038...2024.09.11.1039

<hr /><em>This discussion was created from the release <a href='https://github.com/moneymanagerex/android-money-manager-ex/releases/tag/2024.09.11.1039'>2024.09.11.1039</a>.</em>

**Comment by guanlisheng** [comment]
@sudhegan , can you help to confirm your issue #1819 is solved

**Reply by Sudhegan** [reply]
yes it is solved now thanks!

**Reply by guanlisheng** [reply]
thanks @sudhegan for confirming. 

back to the [article invitation](https://github.com/moneymanagerex/android-money-manager-ex/issues/1819#issuecomment-2337055498), here you go a few references, and sure that you can write up what you want.

1. [MMEX Cloud Storage with NextCloud](https://moneymanagerex.org/blog/mmex-cloud-storage-with-nextcloud/)
2. [Assets - a worked example](https://moneymanagerex.org/blog/assets-a-worked-example/)

**Reply by Sudhegan** [reply]
k i'm ready to write the article how to proceed from here?

**Reply by guanlisheng** [reply]
With markdown on pages, you can fork and create PR after finish in your clones.

also you can post it in our forum or anywhere you are ok and familiar, I can copy and transfer it into markdown on pages. Sure, you are the author.
